charts/alfresco-search-enterprise/values.yaml

```diff
@@ -65,6 +65,12 @@ reindexing:
         # -- Key within the configmap holding the full url to connect to the
         # alfresco repository
         url: REPOSITORY_URL
+  # -- Helm hooks definition
+  policy:
+    # -- When the job will be executed
+    run: post-install
+    # -- When the resources associated with job will be deleted. By default resources will be kept until a new installation.
+    delete: before-hook-creation
```

Comment:
this is the default as per [docs](https://helm.sh/docs/topics/charts_hooks/#hook-deletion-policies), so unless we find it's meaningful to fine tune it, there is no need to have a dedicated value for it